### PR TITLE
Add a Question about how a commit message should be ?

### PR DESCRIPTION
**Pull request by Shaik-mohd-huzaifa**
Checklist:

- [x] I have read [freeCodeCamp's contribution guidelines](https://contribute.freecodecamp.org).
- [x] My pull request has a [descriptive title](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request?id=prepare-a-good-pr-title) (**not** a vague title like `Update index.md`)

Closes #XXXXX

### CODE REVIEW

File: src/data/git-quiz.ts [hunk 1]
```diff
@@ -533,6 +533,16 @@ const gitQuiz = [
...
+  },
+  {
+    Question: "How Should a Git commit message be?",
+    Answer: "It should be short, descriptive and in present tense",
+    Distractor1: "It should be longer",
+    Distractor2: "It should be in oneword",
+    Distractor3: "None of these",
+    Explanation:
+      "A good commit message should be short, descriptive and in present tense.",
+    Link: ""
```
Comment (jdwilkin4) [inline src/data/git-quiz.ts:545]: Please do not leave this field blank
All question submissions need to have a resource

 ```suggestion
    Link: "https://www.freecodecamp.org/news/how-to-write-better-git-commit-messages/"
```

File: src/data/git-quiz.ts [hunk 2]
```diff
@@ -533,6 +533,16 @@ const gitQuiz = [
     Explanation:
       "git commit --amend  is the command used to change the latest commit message.",
     Link: "https://forum.freecodecamp.org/t/git-guide-how-to-amend-your-most-recent-git-commit-message/13186"
+  },
+  {
+    Question: "How Should a Git commit message be?",
+    Answer: "It should be short, descriptive and in present tense",
+    Distractor1: "It should be longer",
+    Distractor2: "It should be in oneword",
```
Comment (jdwilkin4) [inline src/data/git-quiz.ts:541]:
```suggestion
    Distractor2: "It should be in one word",
```

File: src/data/git-quiz.ts [hunk 3]
```diff
@@ -533,6 +533,16 @@ const gitQuiz = [
     Explanation:
       "git commit --amend  is the command used to change the latest commit message.",
     Link: "https://forum.freecodecamp.org/t/git-guide-how-to-amend-your-most-recent-git-commit-message/13186"
+  },
+  {
+    Question: "How Should a Git commit message be?",
```
Comment (jdwilkin4) [inline src/data/git-quiz.ts:538]:
```suggestion
    Question: "In Git, what does a good commit message look like?",
```

File: src/data/git-quiz.ts [hunk 4]
```diff
@@ -533,6 +533,16 @@ const gitQuiz = [
...
+    Answer: "It should be short, descriptive and in present tense",
+    Distractor1: "It should be longer",
```
Comment (jdwilkin4) [inline src/data/git-quiz.ts:540]:
```suggestion
    Distractor1: "It should be long",
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Thank you for your contribution!

Please review the requested changes.
Once those changes have been implemented, then we can merge your PR into main. 👍

**Comment by Shaik-mohd-huzaifa** [comment]
Applied Changes

**Review by jdwilkin4 — APPROVED** [review]
Congrats on another contribution to our repository. 🎉
We look forward to future contributions. 👍

**Pull request merged**